libraries/ESP8266HTTPClient/examples/StreamHttpClient/StreamHttpClient.ino

```diff
@@ -65,7 +67,7 @@ void loop() {
         uint8_t buff[128] = { 0 };
 
         // get tcp stream
```

Comment:
Can we get rid of the `stream` temp variable completely since it's no longer needed with your fix?  i.e. there is no `getStreamPtr()` needed anymore?

Reply:
Actually, I see later on that you still have a "getStreamPtr()" method with the new one, too.  Any reason not to just keep using it?

Reply:
I do not quite understand you here. 
The getStreamPtr() is embedded in the #ifdef KEEP_PRESENT_API, so it's just there for backward compatibility. 
I added the streaming example on purpose, because a server response may be too large to fit into a String, making it impossible to use getString(), but still possible to stream it. Moreover, since it is String getString() instead of String &getString() the response might even be copied (allthough the compiler may optimize this, I don't know)
Could you explain what you are suggesting?

Reply:
I understand you now. The getStreamPtr() is kept as part of the existing API, however in the examples the unneeded call to getStreamPtr() has been removed.